package.json

```diff
@@ -22,11 +22,8 @@
     }
   ],
   "scripts": {
-    "size": "size-limit",
-    "test": "standard && ava",
-    "testci": "standard && ava && npm run size",
-    "prepublish": "npm run check-deps",
-    "check-deps": "nsp check"
+     "test": "standard && nyc ava tap test.js",
```

Comment:
What is the word "tap" doing here?

Reply:
for TAP format report output as console feedback from cmd?

Reply:
however it's not mandatory but just used for report formation.

Reply:
I think you need to specify it differently if you want AVA to output TAP: https://github.com/avajs/ava#tap-reporter